samples/batch_requests/batch_request_with_parsable_as_response_type.py

```diff
@@ -0,0 +1,74 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+#pylint: disable=undefined-variable
+"""Demonstrates using the Batch request with Parsable Resposnse Type"""
+import asyncio
+
+from kiota_abstractions.request_adapter import RequestAdapter
+from kiota_abstractions.request_information import RequestInformation
+from kiota_abstractions.method import Method
+from kiota_abstractions.headers_collection import HeadersCollection as RequestHeaders
+
+from msgraph_core.requests.batch_request_item import BatchRequestItem
+from msgraph_core.requests.batch_request_content import BatchRequestContent
+
+from msgraph_core.requests.batch_request_builder import BatchRequestBuilder
+
+# import User model to serialize to
+from msgraph.generated.models.user import User
+# Create a client
+# code to create graph client
+user_client = GraphServiceClient(credentials=token, scopes=graph_scopes)
+
+print(f"Graph Scopes: {graph_scopes}")
+
+# Create a  request adapter from the client
+request_adapter = user_client.request_adapter
+
+# Create an instance of BatchRequestBuilder
+batch_request_builder = BatchRequestBuilder(request_adapter)
```

Comment:
There's one last step we are missing to tie this al together with the service libraries. See 
https://github.com/microsoftgraph/msgraph-sdk-php/issues/1578https://github.com/microsoftgraph/msgraph-sdk-php/issues/1578

So this should not initliazed by the user. Rather line 62 should be something like this

```python
batch_response_content = await user_client.batch.post(batch_request_content=batch_content)
````